src/chat/services/resource-content.service.ts

```
@@ -412,33 +356,22 @@ export class ResourceContentService {
+    const contentByPath = new Map(hits.map((h) => [h.path, h.content]));
+    const documentCandidates = chunkPaths
+      .map((chunkPath) => {
+        const content = contentByPath.get(
+          this.normalizeResourcePath(chunkPath),
+        );
+        if (!content) return null;
+        const title = chunkPath.split('/').pop() || chunkPath || '문서';
+        return { title, content, path: chunkPath };
+      })
+      .filter(
+        (r): r is { title: string; content: string; path: string } => r !== null,
```

ESLint

[error] Insert `⏎·········`

(prettier/prettier)